build.sbt

```diff
@@ -18,11 +18,18 @@ ThisBuild / githubWorkflowBuild := Seq(
     name = Some("Build native assembly")
   )
 )
-ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8", "graalvm-ce-java11@")
-ThisBuild / githubWorkflowOSes := Seq("ubuntu-latest", "macos-latest")
-ThisBuild / githubWorkflowBuildMatrixExclusions := Seq(MatrixExclude(Map("os" -> "macos-latest", "java" -> "adopt@1.8")))
```

Comment:
It seems we lost this exclusion, does it matter?

Reply:
CI is green, so I think all is good?

Reply:
Given all the pipelines are now passing, I think it shouldn't matter anymore.
TBH I don't even know why we had this, but at last, it's simpler now.

Reply:
Good enough for me :)